clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -291,7 +293,22 @@ public void onEviction(long key) {
 
   @Override
   public void synchronizeKeyToPassive(PassiveSynchronizationChannel<EhcacheEntityMessage> syncChannel, int concurrencyKey) {
-    throw new UnsupportedOperationException("Active/passive is not supported yet");
+    if (concurrencyKey == 1) {
```

Comment:
Please reference `org.ehcache.clustered.server.ConcurrencyStrategies.DefaultConcurrencyStrategy#DEFAULT_KEY` instead of a hard coded value.